### PR TITLE
Added table of German EGA studies. Fixed some typos.

### PR DESCRIPTION
**Pull request by vinjana**
Just get an impression of what I changed. 

The further plan is to get the studies on a separate page and map data.ghga.de on that page. I am not so much in CSS and HTML + Jekyll to be able to say how to do that. Any suggestions would be appreciated!

### CODE REVIEW

File: _includes/studies.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+                <p>
+                    We intend to optimize the procedures and processes involved with the deposition and access to
+                    human datasets. These procedures should be simplified from a technical and organisational
+                    perspective while ensuring full compliance to data protection and safety standards.
+                </p>
+
+                <p>
+                    To this aim a diverse and distributed data infrastructure, involving major data generators and
+                    build using scalable cloud technologies, will have to be integrated and exposed to researcher
```
Comment (KochTobi) [inline _includes/studies.html]: What is meant by 'and build', can this be left out?
Also we should change researcher to researchers plural.

Comment (vinjana) [reply _includes/studies.html]: Probably this should be "built".  The data infrastructure **is** already built using cloud technologies. This refers for instance to the S3 storage already in place in the de.NBI cloud. Therefore it is also "**and** built".

**Comment by vinjana** [comment]
Unless you have serious concerns about the content I suggest to merge this PR for the upcoming remote meeting.

**Comment by vinjana** [comment]
Cool, thanks. Ivo will show the current status in the remote meeting (if we get jekyll and stuff installed on his system).

**Pull request merged**